src/components/atom/Atom.test.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import { render } from "@testing-library/react";
+import cases from "jest-in-case";
+import Atom, { colors, AtomColorVariant, ColorMap } from "./Atom";
+
+cases(
+  "should apply the colors correctly",
+  ({ colorKey }: { colorKey: AtomColorVariant }) => {
+    const { container } = render(<Atom color={colorKey} />);
+
+    const circle = container.querySelector("div > svg > g > circle");
+    const getEllipse = (idx: number) =>
+      container.querySelector(`div > svg > g > ellipse:nth-child(${idx})`);
```

Comment:
As you can't use the usual Testing Library syntax (`getByRole | getByText` etc), could you add a comment outlining what you are selecting here...just so anyone looking at the test can easily understand what is being tested.

Reply:
Sure, good suggestion. I've updated the test with comments 👍 .